src/style.css

```diff
@@ -1 +1,49 @@
-/* add styles here */
+@font-face {
+  font-family: Avenir, sans-serif;
```

Comment:
You should provide a fallback font family in case the main one doesn't work. For example, 'font-family: Avenir, Arial, sans-serif;'. This ensures text is still readable if the custom font fails to load.